apps/condo/domains/acquiring/schema/RecurrentPaymentContext.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Generated by `createschema acquiring.RecurrentPaymentContext 'limit:Text;paymentDay:Integer;settings:Json;billingCategories:Json'`
+ */
+const { Text, Relationship, Integer, Checkbox } = require('@keystonejs/fields')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/RecurrentPaymentContext')
+const { RECURRENT_PAYMENT_CONTEXT_BOTH_TRIGGER_SET_UP_ERROR } = require('@condo/domains/acquiring/constants/errors')
+
+
+const RecurrentPaymentContext = new GQLListSchema('RecurrentPaymentContext', {
+    schemaDoc: 'Recurrent payments context configuration. Holding settings, limits and etc, configured by resident for background process that proceeding resident`s receipts on monthly basis.',
+    fields: {
+        enabled: {
```

Comment:
Sometimes when we needed a checkbox we used a link to the user who last pressed the checkbox. Like: 

instead of having a field `approved: Bool` we have a field `approvedBy: User`

It should speed up a debugging and solving complex cases from support. 

--

Im not in a good context about this task, but maybe this is a good idea

Reply:
What about? We don't need that?